.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -43,17 +43,47 @@ If you need to submit a pr for any of these repos during this period, you should
+
+`git clone git@github.com:adoptium/jdk17u.git`
+
+`cd jdk17u`
+ 
+`git tag -a "jdk-17.0.6-beforereleastest-ga" jdk-17.0.6+8^{} -m"Before release trial test"`
+
+`git push --tags origin master`
+
+  <li>Wait release trigger job to detect the tag (wait up to 10mins), eg: https://ci.adoptopenjdk.net/job/build-scripts/job/utils/job/releaseTrigger_jdk17u/ (if before the 13th day of the month then you will need to manually run the job as it will be outside its cron schedule)
+  <li>The trial release pipeline job should now be running, eg: https://ci.adoptopenjdk.net/job/build-scripts/job/release-openjdk17-pipeline/
+  </ul>
+</details>
+
+- [ ] Confirm successful trial release pipelines and jck completion
+- [ ] Calculate the "expected" openjdk build tags for the releases being published, and update all the JDKnn_BRANCH values in the testenv.properties file for the aqa-tests release branch, eg: https://github.com/adoptium/aqa-tests/blob/v0.9.6-release/testenv/testenv.properties 
+
 -------
 
 Release Week Checklist:
 
 - [ ] **Add website banner** (_automate_* via github workflow in website repository) - Announce that we target releases to be available within 48-72 hours of the GA tags being available
 - [ ] **Check Tags have been released upstream** - Look for mailing list announcements and `-ga` tags in version control.
+- [ ] Check the published GA tags are the "expected" tags entered in the aqa-tests release branch testenv.properties. If they are not then update.
 - [ ] **Check Tags have been Mirrored** [Mirrors](https://ci.adoptopenjdk.net/view/git-mirrors/job/git-mirrors/job/adoptium/).
-- [ ] **Launch build pipelines** for each version being released [(as per release doc](https://github.com/adoptium/temurin-build/blob/master/RELEASING.md#steps-for-every-version)) once release tags are available via [launch page](https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk8-pipeline/build) in Jenkins.  Provide links in this issue to each version's pipeline build(s). There may be multiple pipelines per version if primary and secondary platforms are separated to expedite the release.  In some cases,  where there are unforeseen configuration or infrastructure issues, reruns may be needed.
+- [ ] **Check "auto-trigger" pipelines or Launch build pipelines** for each version being released. Verify if the release pipline "auto-triggered", if not (maybe expected tag was wrong), then manually launch [(as per release doc](https://github.com/adoptium/temurin-build/blob/master/RELEASING.md#steps-for-every-version)) once release tags are available via [launch page](https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk8-pipeline/build) in Jenkins.  Provide links in this issue to each version's pipeline build(s). There may be multiple pipelines per version if primary and secondary platforms are separated to expedite the release.  In some cases,  where there are unforeseen configuration or infrastructure issues, reruns may be needed.
```

Comment:
should we mention: as is written now, jdk8 aarch32 linux wont be build by auto-trigger

Reply:
@zdtsw Can we fix that? Is there a reason it could not be auto-triggered?

Reply:
we could I think, we've just run out of time a bit for this release.

Reply:
Raised: https://github.com/adoptium/temurin-build/issues/3200

Reply:
think the original idea i wanted to exclude this one is, not sure how much resource we will use for jdk8 other platform + other jdk versions.  since jdk8 aarch32  is not the primary platform, we can be more flex to decide when we want to build and test.